### ISSUE TITLE
"Moment Timezone requires Moment.js" error when using Node package

### ISSUE THREAD

**Issue by slaponicus** [moment/moment-timezone#838]
Moment-timezone 0.5.23 throws the following error when attempting to use:

`Moment Timezone requires Moment.js.`

I have attempted to upgrade to the latest package version but it doesn't solve the issue. In researching, I found that it's probably related to https://github.com/moment/moment/issues/5481 since moment is a dependency. 

Would it be possible/a good idea to change the moment version in the package.json file to exclude the versions of moment which cause this issue?

Thanks!

**Comment by simplecommerce** [comment]
I have the same issue here.

**Comment by phillt** [comment]
Broke our page as well:

![image](https://user-images.githubusercontent.com/2586937/80840678-a3658380-8bb2-11ea-8ffd-f575a095d91d.png)

**Comment by voltaaage** [comment]
Broke ours as well 

![Screen Shot 2020-05-01 at 6 23 21 PM](https://user-images.githubusercontent.com/7111237/80851835-dde21700-8bd8-11ea-89d1-a9246ec5efe4.png)

**Comment by btmurrell** [comment]
+1

**Comment by consciousweb** [comment]
+1 anyone know the last version combination that didnt throw errors?

**Comment by Cherviakov** [comment]
Previous issue #837 fix worked for me.

**Comment by laddhadhiraj** [comment]
+1

**Comment by rinick** [comment]
for now just rollback to 2.24
if you need another package that already depends on 2.25, update package.json's resolutions

``` 
  "resolutions": {
    "**/minimist": "^1.2.5",
    "**/moment": "2.24.0"
  },
```

**Comment by ariel262626** [comment]
Setting moment version to "2.24.0" fixed it for me

**Comment by mitskevich** [comment]
Upgrading moment to 2.25.3 fixed it for me. I had the same issue with moment >2.24.0 <=2.25.2 and moment-timezone 0.5.28.

**Comment by onlythinking** [comment]
+1，I also encountered the same problem

**Comment by slaponicus** [comment]
Fixed since moment 2.25.3 got released
